master/docs/_images/Makefile

```diff
@@ -1,5 +1,5 @@
 
-SOURCES = overview_src.svg changes_src.svg workers_src.svg master_src.svg
+SOURCES = overview_src.svg changes_src.svg workers_src.svg master_src.svg multimaster_src.svg auth_rules_src.svg
```

Comment:
I did not understand what are these `_src` files for. Could you explain?

Reply:
src files are for editing with inkscape. It contains additional information needed for e.g. editing of text. It's later exported to svg which contains text outline that can no longer be edited via inkscape.

Reply:
ok thanks